src/webtoon_downloader.py

```diff
@@ -410,17 +507,24 @@ def download_chapter(
     dest:
         destination folder path to store the downloaded image files.
         (default: current working directory)
+
+    export_texts: bool
+        export the chapter title and the author notes into text files.
     """
     log.debug(
         "[italic red]Accessing[/italic red] chapter %d", chapter_info.chapter_number
     )
-    img_urls = get_img_urls(
-        session=session,
-        viewer_url=viewer_url,
-        data_episode_num=chapter_info.data_episode_no,
-    )
+    html = get_chapter_html(session, viewer_url, chapter_info.data_episode_no)
+    soup = BeautifulSoup(html, "lxml")
+    img_urls = extract_img_urls(soup)
     if not os.path.exists(dest):
         os.makedirs(dest)
+    if export_texts:
+        prefix = os.path.join(dest, f"{chapter_info.chapter_number:0{zeros}d}_")
+        with open(prefix + "title.txt", "w") as f:
+            f.write(get_chapter_title(soup) + '\n')
+        with open(prefix + "notes.txt", "w") as f:
+            f.write(get_chapter_notes(soup) + '\n')
```

Comment:
I would personally prefer that all of the chapter metadata be extracted to a json file, which are easier to handle, for the following reasons:
- Some of the text files might be empty 
- Easily be able to extend the file with more data in the future, apart from `title` and `notes`

**But** this I can see how text file can be easier to reason about as well, so I still think this is great and am fine with it

Reply:
Added an option `--export-format <all|json|text>` to allow selecting the output format

Reply:
Did you meant `export-format` or `text-format` ? Both are fine to me.

Reply:
Good point, the code didn't match the commit message. Now changed the code to also use the `--export-format` name: it makes more sense to me.